package.json

```diff
@@ -22,16 +22,20 @@
     "dotenv": "^16.0.2",
     "ejs": "^3.1.8",
     "express": "~4.16.1",
+    "express-session": "^1.17.3",
     "google-auth-library": "^8.6.0",
     "helmet": "^6.0.0",
     "http-errors": "~1.6.3",
     "joi": "^17.6.4",
     "jsonwebtoken": "^8.5.1",
+    "memorystore": "^1.6.7",
```

Comment:
Why Install Memorystore ?

Reply:
![session](https://user-images.githubusercontent.com/49001006/206639052-12593c2c-ced6-4dcd-8c9b-a28ab2f78bf5.png)

From express-session middleware documentation, the default MemoryStore is not fit for a production environment.

![memorystore](https://user-images.githubusercontent.com/49001006/206639632-840e00a2-ee09-41a7-be1d-96583e9c1e00.png)

There are other recommended compatible modules for session store but the memorystore module pretty much works well also for a production use case.